src/v2/Apps/Fair/FairApp.tsx

```diff
@@ -72,25 +66,6 @@ const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
 
       <FairHeaderFragmentContainer fair={fair} />
 
-      {hasArticles && (
-        <Box my={6} pt={4} borderTop="1px solid" borderColor="black10">
-          <Box display="flex" justifyContent="space-between">
-            <Text variant="lg" as="h3" mb={2}>
-              Related Reading
-            </Text>
-
-            {/* @ts-expect-error STRICT_NULL_CHECK */}
-            {fair.articlesConnection.totalCount > FAIR_EDITORIAL_AMOUNT && (
-              <RouterLink to={`${fair.href}/articles`} noUnderline>
-                <Text variant="sm">View all</Text>
-              </RouterLink>
-            )}
-          </Box>
-
-          <FairEditorialFragmentContainer fair={fair} />
-        </Box>
-      )}
```

Comment:
Note, I didn't delete the actual components because perhaps we might want to add this back in a more visually elegant way later?